tests/data/small_sky_xmatch_margin/catalog_info.json

```diff
@@ -0,0 +1,7 @@
+{
+    "catalog_name": "small_sky_xmatch_margin",
+    "catalog_type": "margin",
+    "total_rows": 26,
+    "primary_catalog": "small_sky_xmatch",
```

Comment:
Is this based off the `small_sky` or `small_sky_order1` catalog? I think it would make more sense to use one of the primary catalogs from the hipscat repository, rather than from the lsdb repo.

Can you add a small README in this directory explaining how to generate the data, and how it's used? a la https://github.com/astronomy-commons/hipscat/blob/main/tests/data/small_sky_source_object_index/README.md

Reply:
It's based off the small_sky_order1, with the points randomly moved a little. It's a good point it should be one of the hispcat catalogs, I just had this one already from testing lsdb stuff, but yeah I'll generate one for the hipscat catalogs.